modules/edu.gemini.epics.acm/src/main/java/edu/gemini/epics/acm/CaObserveSenderImpl.java

```diff
@@ -25,8 +25,7 @@
  */
 public class CaObserveSenderImpl implements CaApplySender {
```

Comment:
Just a general question. Would it be worth converting these to scala?

Reply:
I don't know. At some point we have to deal with Java code, because of JCA.